index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/* eslint-disable no-underscore-dangle */
+class CoverageBase {
+    /** Constructor for Coverage plugin
+     * @method Constructor
+     * @param {Object}  config  Configuration
+     * @return {CoverageBase}
+     */
+    constructor(config) {
+        this.config = config;
+    }
+
+    /**
+     * Get shell command to upload coverage to server
+     * @method getUploadCoverageCmd
+     * @return {Promise}     Shell commands to upload coverage
+     */
+    getUploadCoverageCmd() {
+        return this._uploadCoverage();
+    }
+
+    _uploadCoverage() {
```

Comment:
not sure if this name should match and be `_getUploadCoverageCmd`

Reply:
loll it should, good catch!